test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -1030,17 +1031,16 @@ describe('the rubicon adapter', () => {
   describe('user sync', () => {
     let bids;
     let server;
-    let clock;
     let addBidResponseAction;
     let rubiconAdapter;
     const emilyUrl = 'https://tap-secure.rubiconproject.com/partner/scripts/rubicon/emily.html?rtb_ext=1';
-    let origGetConfig = window.$$PREBID_GLOBAL$$.getConfig;
 
     beforeEach(() => {
       bids = [];
 
       server = sinon.fakeServer.create();
-      clock = sinon.useFakeTimers();
+      // monitor userSync registrations
+      sinon.spy(userSync, 'registerSync');
```

Comment:
I think this needs to be a stub?

Spies only observe things... so I think all the `registerSync` calls make real mutations to the real usersync global state.

Tests shouldn't have side effects like this.